site/gatsby-site/blog/incident-report-2022-october/index.fr.mdx

```diff
@@ -1,143 +1,142 @@
+En dehors de l'utilisation délibérée des systèmes d'IA pour collecter et utiliser des données privées, il existe plusieurs exemples antérieurs de systèmes automatisés collectant ou partageant par erreur ces données. En 2018, un Amazon Echo a envoyé par erreur une conversation privée enregistrée entre un mari et sa femme à l'un des employés du mari à leur insu. GPT-2, le prédécesseur de GPT-3, aurait été capable de réciter des informations personnelles identifiables (PII) qu'il a apprises grâce à une formation sur des quantités massives de données provenant d'Internet.
 
-These occurrences from the last few years highlight common themes of privacy concerns that are increasingly concerns of legal systems providing rights to privacy and data protection.
+Ces événements de ces dernières années mettent en évidence des thèmes communs de préoccupations en matière de vie privée qui concernent de plus en plus les systèmes juridiques garantissant des droits à la vie privée et à la protection des données.
 
-_– by Janet Schwartz_
+_- par Janet Schwartz_
 
-## 👇 Diving Deeper
+## 👇 Plonger plus profondément
 
-* All new incidents _added_ to the database in the last month, grouped by topic:
-    - **Privacy & surveillance**: [#354](/cite/354), [#357](/cite/357), [#360](/cite/360), [#361](/cite/361), [#371](/cite/371), [#372](/cite/372), [#377](/cite/377)
-    - **Facial recognition:** [#358](/cite/358), [#365](/cite/365), [#368](/cite/368), [#373](/cite/373), [#375](/cite/375), [#385](/cite/385)
-    - **Bias & discrimination:** [#355](/cite/355), [#356](/cite/356), [#359](/cite/359), [#367](/cite/367), [#374](/cite/374), [#383](/cite/383/)
-    - **Social media:** [#362](/cite/362), [#363](/cite/363), [#366](/cite/366), [#380](/cite/380), [#382](/cite/382)
-    - **Impactful errors:** [#364](/cite/364), [#379](/cite/379), [#384](/cite/384)
-    - **Unfair competition:** [#370](/cite/370), [#376](/cite/376)
-    - **Autonomous vehicles:** [#378](/cite/378), [#381](/cite/381)
-    - **Labor displacement:** [#369](/cite/369)
-* Explore clusters of similar incidents in [Spatial Visualization](/summaries/spatial)
-* Check out [Table View](/apps/incidents) for a complete view of all incidents
-* Learn about alleged developers, deployers, and harmed parties in [Entities Page](/entities)
+* Tous les nouveaux incidents _ajoutés_ à la base de données au cours du mois dernier, regroupés par sujet :
+    - **Confidentialité et surveillance** : [#354](/cite/354), [#357](/cite/357), [#360](/cite/360), [#361](/cite/ 361), [#371](/cite/371), [#372](/cite/372), [#377](/cite/377)
+    - **Reconnaissance faciale :** [#358](/cite/358), [#365](/cite/365), [#368](/cite/368), [#373](/cite/373 ), [#375](/cite/375), [#385](/cite/385)
+    - **Biais et discrimination :** [#355](/cite/355), [#356](/cite/356), [#359](/cite/359), [#367](/cite/ 367), [#374](/cite/374), [#383](/cite/383/)
```

Comment:
There's an extra space in `/cite/ 361` and `[#367](/cite/ 367)` that's causing those links not to work.
![image](https://user-images.githubusercontent.com/25443411/202231914-becfda0c-1a88-4128-930d-245e2a1922b9.png)